src/EPPlus/ConditionalFormatting/ExcelConditionalFormattingConstants.cs

```diff
@@ -263,6 +264,13 @@ internal class Colors
       internal static readonly Color CfvoMiddleValue = Color.FromArgb(0xFF,0xFF,0xEB,0x84);
       internal static readonly Color CfvoHighValue = Color.FromArgb(0xFF,0x63,0xBE,0x7B);
     }
-    #endregion Colors
-  }
+
+        internal class DxfColor
```

Comment:
Seems empty, remove?

Reply:
Yep. Also found a bunch of other references that only lead to classes that are not used anymore so I'm removing those too